src/App.jsx

```diff
@@ -229,6 +238,26 @@ class App extends Component {
       [add.USDT, usdt.interface.encodeFunctionData('balanceOf', [add.MCD_JOIN_USDT_A])], // 86
       [add.MCD_FLIP_USDT_A, usdtAFlip.interface.encodeFunctionData('kicks', [])], // 87
 
+      [add.MCD_VAT, vat.interface.encodeFunctionData('ilks', [compAIlkBytes])], // 88
+      [add.MCD_JUG, jug.interface.encodeFunctionData('ilks', [compAIlkBytes])],
+      [add.MCD_SPOT, spot.interface.encodeFunctionData('ilks', [compAIlkBytes])], // 90
+      [add.COMP, comp.interface.encodeFunctionData('totalSupply', [])],
+      [add.COMP, comp.interface.encodeFunctionData('balanceOf', [add.MCD_JOIN_COMP_A])], // 92
+      [add.MCD_FLIP_COMP_A, compAFlip.interface.encodeFunctionData('kicks', [])], // 93
+
+      [add.MCD_VAT, vat.interface.encodeFunctionData('ilks', [lrcAIlkBytes])], // 94
+      [add.MCD_JUG, jug.interface.encodeFunctionData('ilks', [lrcAIlkBytes])],
+      [add.MCD_SPOT, spot.interface.encodeFunctionData('ilks', [lrcAIlkBytes])], // 96
+      [add.LRC, lrc.interface.encodeFunctionData('totalSupply', [])],
+      [add.LRC, lrc.interface.encodeFunctionData('balanceOf', [add.MCD_JOIN_LRC_A])], // 98
+      [add.MCD_FLIP_COMP_A, lrcAFlip.interface.encodeFunctionData('kicks', [])], // 99
```

Comment:
For sure this doesn't look right. 
I think it should be `add.MCD_FLIP_LRC_A,` @nanexcool